Aquality.Selenium/tests/Aquality.Selenium.Tests/Integration/BrowserTabsTests.cs

```diff
@@ -148,16 +148,5 @@ private void CheckSwitchingBy(int expectedTabCount, Action switchMethod)
             Assert.AreEqual(newTabHandle, browser.Tabs().CurrentTabHandle, "Browser should be switched to correct tab");
             Assert.AreEqual(expectedTabCount, browser.Tabs().TabHandles.Count, "Number of tabs should be correct");
         }
-
-        private void CheckSwitching(int expectedTabCount, Action switchMethod)
```

Comment:
why was this test removed?

Reply:
that was not a test but a private unused method, it was never used actually